### PR TITLE
[PLAYER-3626] Add showing data on Chromecast view  when the second devise cast different video

### PR DESCRIPTION
**Pull request by atopilskii**
Add showing data on Chromecast view  when the second device is connected
there is no new test because we cannot cover it by test
all old tests are passed

### CODE REVIEW

File: ChromecastSampleApp/app/src/main/java/com/ooyala/sample/ChromecastPlayerActivity.java [hunk 1]
```diff
@@ -155,6 +158,19 @@ public void update(Observable arg0, Object argN) {
     Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
   }
 
+  private void updateCustView(OoyalaNotification notification) {
+    VideoData data = (VideoData) notification.getData();
```
Comment (IvanSakharovskii) [inline ChromecastSampleApp/app/src/main/java/com/ooyala/sample/ChromecastPlayerActivity.java]: check that data is `VideoData`

File: ChromecastSampleApp/app/src/main/java/com/ooyala/sample/CastViewManager.java [hunk 2]
```diff
@@ -21,17 +21,17 @@
     stateTextView = castView.findViewById(R.id.castStateTextView);
   }
 
-  void configureCastView(Video video) {
+  void configureCastView(String title, String description, String imageUrl) {
     final ImageView castBackgroundImage = castView.findViewById(R.id.castBackgroundImage);
 
     // Update the ImageView on a separate thread
-    new Thread(new UpdateImageViewRunnable(castBackgroundImage, video.getPromoImageURL(0, 0))).start();
+    new Thread(new UpdateImageViewRunnable(castBackgroundImage, imageUrl)).start();
```
Comment (IvanSakharovskii) [inline ChromecastSampleApp/app/src/main/java/com/ooyala/sample/CastViewManager.java]: Why `new Thread`, I suppose you can just call `new UpdateImageViewRunnable().run()`

File: ChromecastSampleApp/app/src/main/java/com/ooyala/sample/ChromecastPlayerActivity.java (hunk 1 above)
Comment (atopilskii) [reply ChromecastSampleApp/app/src/main/java/com/ooyala/sample/ChromecastPlayerActivity.java]: fixed

**Review by akshaydch11 — APPROVED** [review]

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**